src/appshell/qml/Preferences/GeneralPreferencesPage.qml

```diff
@@ -42,6 +42,10 @@ PreferencesPage {
         }
     }
 
+    ProgrammeStartPreferencesModel {
```

Comment:
This model looks a bit redundant now. You can move its implementation into GeneralPreferencesModel